component/pom.xml

```diff
@@ -11,29 +11,29 @@
 
     <artifactId>component</artifactId>
     <dependencies>
```

Comment:
You have utilised parent pom.xml for lombok and java as per review from iluwatar, this looks good - still works